### PR TITLE
community: fix issue of the existence of numeric object in `additional_kwargs` a…

### PR DESCRIPTION
**Pull request by Dobiichi-Origami**
- **Description:** A previous PR breaks the code from `baidu_qianfan_endpoint.py` which causes the malfunction of streaming

### CODE REVIEW

**Comment by Dobiichi-Origami** [comment]
PTAL @efriis

**Comment by Dobiichi-Origami** [comment]
@ccurme

File: libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py [hunk 1]
```diff
@@ -613,6 +617,15 @@ def _stream(
                         role="assistant",
                         additional_kwargs=additional_kwargs,
                         usage_metadata=msg.usage_metadata,
+                        tool_call_chunks=[
```
Comment (ccurme) [inline libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py:620]: Should we add this to `_astream` below, as well?

**Review by ccurme — APPROVED** [review]
Thank you

File: libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py (hunk 1 above)
Comment (Dobiichi-Origami) [reply libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py:620]: oh, I forgot that

**Pull request merged**